lib/helpers.js

```diff
@@ -149,9 +193,20 @@ _beforeEach.givenUserWithRole = function (attrs, role, optionalHandler) {
       }
 
       test.userRole = result;
-      test.app.models.roleMapping.create(
+
+      var roleMappingModelName = test.roleMappingModel ?
+        test.roleMappingModel : DEFAULT_ROLE_MAPPING_MODEL;
+
+      var RoleMapping = test.app.models[roleMappingModelName];
+
+      if (!RoleMapping) {
+        return done('RoleMapping model with name ' +
+          roleMappingModelName + ' not found');
+      }
+
+      RoleMapping.create(
         {principalId: test.user.id,
```

Comment:
The **User** model name also needs to be dynamically resolved. So, `test.user.id` will not resolve. Something like below would work:
```javascript
var userModelName = this.userModel ?  this.userModel : DEFAULT_USER_MODEL;
RoleMapping.create(
    {principalId: test[userModelName].id,
    principalType: RoleMapping.USER,
    roleId: result.id}, ..)
```